docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md

```diff
@@ -65,7 +65,7 @@ showAsideToc: true
 
 </ul>
 
-You can configure the on-premises cloud provider for YugabyteDB using the Yugabyte Platform UI. If no cloud providers are configured, the main Dashboard prompts you to configure at least one cloud provider.
+You can configure the on-premises cloud provider for YugabyteDB using the Yugabyte Platform UI. If no cloud providers are configured, the main Dashboard prompts you to configure at least one cloud provider, as per the following illustration:
```

Comment:
Yugabyte Platform UI - changed to console elsewhere

Reply:
Changed